packages/react/src/index.ts

```diff
@@ -1 +1,42 @@
-export * from './lib/react'
+import '@digdir/designsystemet-css'
+
+export * from './react-types.d'
```

Comment:
Maybe consider this syntax for component files too?

Reply:
Discussed that it is probably better to keep them separated